core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -117,11 +117,28 @@ class KafkaApis(val requestChannel: RequestChannel,
         case ApiKeys.DELETE_RECORDS => handleDeleteRecordsRequest(request)
         case ApiKeys.INIT_PRODUCER_ID => handleInitProducerIdRequest(request)
         case ApiKeys.OFFSET_FOR_LEADER_EPOCH => handleOffsetForLeaderEpochRequest(request)
-        case ApiKeys.ADD_PARTITIONS_TO_TXN => handleAddPartitionToTxnRequest(request)
-        case ApiKeys.ADD_OFFSETS_TO_TXN => handleAddOffsetsToTxnRequest(request)
-        case ApiKeys.END_TXN => handleEndTxnRequest(request)
-        case ApiKeys.WRITE_TXN_MARKERS => handleWriteTxnMarkersRequest(request)
-        case ApiKeys.TXN_OFFSET_COMMIT => handleTxnOffsetCommitRequest(request)
+        case ApiKeys.ADD_PARTITIONS_TO_TXN => handleWithInterBrokerVersionAtLeast(KAFKA_0_11_0_IV0, request, (error:Errors) => {
```

Comment:
I've done the first part, but i don't get what i'm supposed to do for the second part? We have `ApiVersion` which is what i'm using to determine if we have `inter.broker.protocol.version` set to at least `KAFKA_0_11_0_IV0`. We then have `ApiVersionsResponse` that uses `ApiKeys.values()` to build a list of an inner class `ApiVersion`
How do we tie an `ApiVersion` as in `KAFKA_0_11_0_IV0` to the list of supported operations?